ui/panels-plugin/src/plugins/gauge-chart/GaugeChartOptionsEditor.tsx

```diff
@@ -39,19 +32,5 @@ export function GaugeChartOptionsEditor(props: GaugeChartOptionsEditorProps) {
     );
   };
 
-  return (
-    <OptionsEditorTabs
-      tabs={{
-        query: {
-          content: <TimeSeriesQueryEditor value={query} onChange={handleQueryChange} />,
-        },
-        settings: {
-          content: <GaugeChartOptionsEditorSettings {...props} />,
-        },
-        json: {
-          content: <JSONEditor {...props} />,
-        },
-      }}
-    />
-  );
+  return <TimeSeriesQueryEditor value={query} onChange={handleQueryChange} />;
```

Comment:
The naming of these components is kind of confusing now, for instance this is still called GaugeChartOptionsEditor but it only returns the query editor

Reply:
renamed these components to xxxChartQueryEditor instead